source/ClusterAPI/Setup.rst

```diff
@@ -12,16 +12,6 @@ Setup a new cluster using Cluster API
 Deployment Considerations
 =========================
 
-Deployment Account
-------------------
-
-The deployment is tied to the account which created it. This includes their
-credentials being encoded as an opaque secret within the cluster for the cluster API openstack component.
-
-For production workloads, and long-lived clusters with multiple accounts a service account it recommended.
-This means the cluster isn't tied to the lifetime of a single user's account or password. A service
-account can be requested through our :ref:`contact_details`.
-
```

Comment:
I think we should keep the original CAPI docs alongside the new docs for now - maybe keep the original docs in a separate page?

Reply:
I would argue not, it's in source control and I'm not keen to support this method at all going forwards

I'm happy to talk about it offline though if you feel strongly :slightly_smiling_face: